dags/analyzer_helper/telegram/tests/unit/test_transform_raw_data.py

```
@@ -0,0 +1,635 @@
+                    },
+                    {
+                        "reaction": '[{"type":"emoji","emoji":"🙌"}]',
+                        "reaction_date": 1713038349.0,
+                        "reactor_id": 2.0,
+                    },
+                ],
+                "replies": [],
+                "mentions": [],
+            }
+        ]
+        result = self.transformer.transform(data)
```

Ruff

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)